examples/step-37/step-37.cc

```diff
@@ -774,283 +874,247 @@ namespace Step37
                  << time() << "s/" << time.wall_time() << "s" << std::endl;
     time.restart();
 
-    system_matrix.reinit (dof_handler, constraints);
-    std::cout.precision(4);
-    std::cout << "System matrix memory consumption:     "
-              << system_matrix.memory_consumption()*1e-6
-              << " MB."
-              << std::endl;
+    {
+      typename MatrixFree<dim,double>::AdditionalData additional_data;
+      additional_data.tasks_parallel_scheme =
+        MatrixFree<dim,double>::AdditionalData::none;
```

Comment:
It is technically easy to add multithreading, yes, but I want to show that in step-48 because mixing MPI and threads is a new topic. There is enough new content in this tutorial already. :-)